src/main/java/org/fundacionjala/virtualassistant/clients/openai/OpenAIClient.java

```diff
@@ -0,0 +1,42 @@
+package org.fundacionjala.virtualassistant.clients.openai;
+
+import com.theokanning.openai.completion.*;
+import com.theokanning.openai.service.OpenAiService;
+
+import io.github.cdimascio.dotenv.Dotenv;
+
+public class OpenAIClient implements AIClientInterface {
+    private static final Dotenv dotenv = Dotenv.load();
+    private static final String TOKEN = dotenv.get("TOKEN_AICLIENTE");
+    public static final String MODEL = "text-davinci-003";
+    public static final int MAX_TOKENS = 1000;
+    public static final double TEMPERATURE = 0.8;
+    public static final boolean ECHO = true;
+    private final OpenAiService service;
+
+    public OpenAIClient(OpenAiService service) {
+        this.service = service;
+    }
+
+    @Override
+    public String chat(String request) {
+        OpenAiService service = new OpenAiService(TOKEN);
```

Comment:
why dont use here the service that you have already created, why you are instantiating it again?